styles.css

```diff
@@ -0,0 +1,25 @@
+body{
+    font-family: 'Kaushan Script', cursive;
+}
+h1{
+    font-size: 60px;
+}
+h2{
+    font-size: 30px;
+}
+p{
+    font-size: 18px;
+}
+h1{
+    font-weight: bold;
+}
+h1{
+    text-decoration: underline;
+}
+body{
+    text-align: center;
+}
+body{
+    background-color: rgb(255, 237, 159);
+    color: rgb(0, 89, 138);
+}
```

Comment:
You should combine all your `body` selectors into one CSS ruleset, these 2 instances can be incorporated to the one on lines 1-3.